runtime/elem/builtins/Core.h

```diff
@@ -19,28 +20,20 @@ namespace elem
             return channelIndex.load();
         }
 
-        FloatType getTargetGain()
-        {
-            return targetGain.load();
-        }
-
         bool active()
         {
-            return targetGain.load() > 0.0f;
+            return fadeOut.targetGain > 0.5;
         }
 
         bool stillRunning()
         {
-            auto const t = targetGain.load();
-            auto const c = currentGain.load();
-
-            return (t >= 0.5 || (std::abs(c - t) >= 1e-6));
+            return active() || !fadeOut.settled();
         }
 
-        void activate(FloatType initialGain = FloatType(0)) 
+        void setFades(FloatType fadeInMs, FloatType fadeOutMs) 
```

Comment:
This new method goes away too if we use the setProperty chain, which I find cleaner

Reply:
💯